src/dp_abstraction/aca_goal_state_handler.cpp

```diff
@@ -190,6 +190,9 @@ void Aca_Goal_State_Handler::add_goal_state_operation_status(
 
   if (operation_rc == EXIT_SUCCESS)
     overall_operation_status = OperationStatus::SUCCESS;
+  else if (operation_rc == EINPROGRESS && resource_type == PORT &&
```

Comment:
what is EINPROGRESS btw?

Reply:
/* Operation now in progress */ - that the closest code I can find to represent our "pending" status.